ospd/ospd.py

```diff
@@ -1279,16 +1297,34 @@ def get_scan_end_time(self, scan_id: str) -> str:
 
     def add_scan_log(
```

Comment:
Same for add_scan_log. It would be better to add a second method instead of adding an parameter here.